res/qml/LibraryControl.qml

```diff
@@ -0,0 +1,188 @@
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+
+Item {
+    id: root
+
+    property alias focusWidget: focusedWidgetControl.value
+
+    signal moveSelection(int offset)
+    signal loadSelectedTrack(string group, bool play)
+    signal loadSelectedTrackIntoNextAvailableDeck(bool play)
+
+    FocusedWidgetControl {
+        id: focusedWidgetControl
+
+        Component.onCompleted: this.value = FocusedWidgetControl.WidgetKind.LibraryView
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Library]"
+        key: "GoToItem"
+        onValueChanged: {
+            if (value != 0 && root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView)
+                root.loadSelectedTrackIntoNextAvailableDeck(false);
+
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "LoadSelectedIntoFirstStopped"
+        onValueChanged: {
+            if (value != 0 && root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView)
+                root.loadSelectedTrackIntoNextAvailableDeck(false);
+
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "SelectTrackKnob"
+        onValueChanged: {
+            if (value != 0) {
+                root.focusWidget = 3;
```

Comment:
shouldn't this also use  `FocusedWidgetControl.WidgetKind`?

Reply:
Yes, I forgot to replace this.